src/init.cpp

```diff
@@ -140,16 +144,24 @@ static void init_vulkan()
             NULL                                                        // pUserData
     };
 
-    CVK_VK_GET_INSTANCE_PROC(vkCreateDebugReportCallbackEXT);
 
-    res = fnvkCreateDebugReportCallbackEXT(gVkInstance, &callbackInfo, nullptr, &gVkDebugCallback);
-    CVK_VK_CHECK_FATAL(res, "Can't setup debug callback");
+    if (gDebugReportEnabled) {
+        CVK_VK_GET_INSTANCE_PROC(vkCreateDebugReportCallbackEXT);
+
+        res = fnvkCreateDebugReportCallbackEXT(gVkInstance, &callbackInfo, nullptr, &gVkDebugCallback);
+        CVK_VK_CHECK_FATAL(res, "Can't setup debug callback");
+    }
+    else {
+        cvk_info("VK_EXT_debug_report not enabled");
```

Comment:
I'm half tempted to suggest this should be a warning.

Reply:
Changed to warning.

Reply:
Thanks (wasn't really requesting it TBH, merely thinking out loud)